src/example-types/terms_stats.js

```diff
@@ -2,11 +2,25 @@ var _ = require('lodash/fp')
 var esTwoLevel = require('./esTwoLevelAggregation').result
 let { buildRegexQueryForWords } = require('../regex')
 let { getField } = require('../fields')
+let { metrics, hasValidMetrics } = require('../aggUtils')
 
 module.exports = {
   validContext: context => context.key_field && context.value_field,
   async result(context, search, schema) {
     let field = getField(schema, context.key_field, context.fieldMode)
+    let orderPaths = _.reduce(
```

Comment:
I wonder if `_.mergeWith` would be better here since it seem all we need is to merge `metrics` with an object.